File: ac.js
function ac(amplitude=110,frequency=60,phase=3,phaseDegrees=120){

  const timeStamp = Date.now()
  const positionOfSecond = (timeStamp / 1000) - parseInt(timeStamp / 1000)
  
  const phaseRadians = phaseDegrees * (Math.PI / 180)

  const points = []

  for(phase of new Array(phase).keys()){
    points.push(amplitude * (Math.sin(2 * Math.PI * positionOfSecond * frequency + (phase * phaseRadians))))
  }

  return points
  
}
const canvas = document.getElementById("canvas")

canvas.width = "1000"
canvas.height = "1000"

const startY = 500

const ctx = canvas.getContext('2d')
ctx.strokeStyle = "#000100"

ctx.beginPath()
ctx.lineWidth = "3"

const drawWave = (prevX, ms) => {
  const [y] = ac(100, 1, 1)
  const x = prevX + 2
  ctx.lineTo(x, startY - y)
  ctx.stroke()
  if(x > 1000) return
  setTimeout(() => drawWave(x), 0)
}

drawWave(0)

File: ac.py
import numpy as np
from time import time

def ac(amplitude=110,frequency=60,seconds=2,phases=3,phase_degrees=120):
    
    time_stamp = time()
    second_position = time_stamp / 1000 - int(time_stamp / 1000)

    phase_radians = phase_degrees * (np.pi/180)

    points = []

    for phase in range(phases):
        points.append(amplitude * (np.sin(2 * np.pi * second_position * frequency + (phase * phase_radians))))
    
    return points
